### ISSUE TITLE
Docs for torch.nn.MSELoss are confusing

### ISSUE THREAD

**Issue by blindjesse** [pytorch/pytorch#69984]
### 📚 The doc issue

The documentation for the `reduce` parameter [says the default is `True` ](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/loss.py#L503 )but the function signature [says it's `None`](https://github.com/pytorch/pytorch/blob/c6bcfb152da93fc29e04c6dd32bfe5a311140711/torch/nn/modules/loss.py#L525). The documentation likely needs to be updated

### Suggest a potential alternative/fix

The default value for the reduce parameter should be changed to `None` in the documentation.

**Comment by albanD** [comment]
Note that the "reduce" argument is deprecated. So this is relatively low priority.
